### PR TITLE
75 profile

### LINKED ISSUE #75: Profile
- [ ] 8. A user should be able to enter their name.
- [ ] 9. A user should be able to choose an avatar (emoji animal + color) from a predefined list.
- [ ] 10. Chosen avatars should not be selectable by other users in the household.
- [ ] 11. The avatar should be used in the app to show what the user has done.

### PR DESCRIPTION
**Pull request by ghostworker94**

### CODE REVIEW

File: store/userToHousehold/slice.ts [hunk 1]
```diff
@@ -47,6 +47,66 @@ export const fetchUsersToHouseholds = createAppAsyncThunk<
...
+export const updateNickname = createAppAsyncThunk<
+  UserToHousehold,
+  { nickname: string; userId: number }
+>(
+  'usersToHouseholds/updateNickname',
+  async ({ nickname, userId }, { rejectWithValue }) => {
+    try {
+      const { data: updatedUserToHousehold, error } = await supabase
+        .from('user_to_household')
+        .update({ nickname: nickname })
```
Comment (88jonhan) [inline store/userToHousehold/slice.ts]: i think we have to match with the correct household as well, if the user belongs to multiple households

File: store/userToHousehold/slice.ts [hunk 2]
```diff
@@ -47,6 +47,66 @@ export const fetchUsersToHouseholds = createAppAsyncThunk<
...
+export const updateAvatarEmoji = createAppAsyncThunk<
+  UserToHousehold,
+  { avatarId: number; userId: number }
+>(
+  'usersToHouseholds/updateAvatarEmoji',
+  async ({ avatarId, userId }, { rejectWithValue }) => {
+    try {
+      const { data: updatedUserToHousehold, error } = await supabase
+        .from('user_to_household')
+        .update({ avatar_id: avatarId })
```
Comment (88jonhan) [inline store/userToHousehold/slice.ts:72]: i think we have to match with the correct household as well, if the user belongs to multiple households

File: screens/ProfileScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+              backgroundColor: avatar.colour_code,
+              borderRadius: 50,
+            }}
+          >
+            <Text style={{ fontSize: choosenAvatar === avatar.id ? 60 : 30 }}>
+              {avatar.emoji}
+            </Text>
+          </TouchableOpacity>
+        ))}
+        <Button
```
Comment (88jonhan) [inline screens/ProfileScreen.tsx:339]: cant seem to scroll down and see this button, so I cant submit the new avatar

File: store/auth/slice.ts [hunk 4]
```diff
@@ -77,6 +77,7 @@ export const logoutUser = createAsyncThunk(
     }
   },
 );
+//TODO:  set current user to null, navigate to first scrren
```
Comment (alexjoelaid) [inline store/auth/slice.ts:80]: old unnecessary todo or should we leave it?

**Review by alexjoelaid — APPROVED** [review]

**Pull request merged**